flink-core/src/main/java/org/apache/flink/core/memory/MemoryUtils.java

```diff
@@ -60,32 +63,31 @@
 		}
 	}
 
-	/** Should not be instantiated. */
-	private MemoryUtils() {}
-
-	private static Constructor<? extends ByteBuffer> getDirectBufferPrivateConstructor() {
+	private static long getClassFieldOffset(@SuppressWarnings("SameParameterValue") Class<?> cl, String fieldName) {
+		final String errorMessage = "Could not get field '" + fieldName + "' offset in class '" + cl + "' for unsafe operations";
```

Comment:
This for readability to have it once, otherwise it cannot be shared by separate catch block scopes.
I would hope that JVM will execute it only in case of error anyways.